Include/objimpl.h

```diff
@@ -95,24 +95,9 @@ PyObject_{New, NewVar, Del}.
    the raw memory.
 */
 PyAPI_FUNC(void *) PyObject_Malloc(size_t size);
-#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x03050000
-PyAPI_FUNC(void *) PyObject_Calloc(size_t nelem, size_t elsize);
```

Comment:
It is in the stable API (since version 3.5). It should not be moved.

The same is for other functions, added to the stable API after 3.2.